timesketch/ui/graphs/graph-view.component.ts

```diff
@@ -31,12 +31,56 @@ export class GraphViewComponent {
 
   constructor(private readonly changeDetectorRef: ChangeDetectorRef) {}
 
+  showNeighborhood(event) {
+    let neighborhoodCollection = event.cy.collection()
```

Comment:
Consider naming it just `neighborhood`. If you want to provide type information, add type annotation instead of suffixing the variable. Here the `event.cy.collection()` is sufficient to inform the reader that it is some kind of collection.

Reply:
Relevant: `go/tott-431`.

Reply:
Done.